Makefile

```diff
@@ -1,217 +1,144 @@
-endif
-ifeq ($(shell docker inspect --format="{{ .State.Running }}" $(COMPOSE_PROJECT_NAME)_mail 2> /dev/null),true)
-	@echo Mailhog http://$(shell docker inspect --format='{{(index .NetworkSettings.Networks "$(COMPOSE_NET_NAME)").IPAddress}}' $(COMPOSE_PROJECT_NAME)_mail):8025
-endif
-ifeq ($(shell docker inspect --format="{{ .State.Running }}" $(COMPOSE_PROJECT_NAME)_adminer 2> /dev/null),true)
-	@echo Adminer http://$(shell docker inspect --format='{{(index .NetworkSettings.Networks "$(COMPOSE_NET_NAME)").IPAddress}}' $(COMPOSE_PROJECT_NAME)_adminer)
-endif
-
-chown:
-# Use this goal to set permissions in docker container
-	$(call php-0, /bin/sh -c "chown $(CUID):$(CGID) /var/www/html/web -R")
-# Need this to fix files folder
-	$(call php-0, /bin/sh -c "chown www-data: /var/www/html/web/sites/default/files -R")
+	$(info Containers for "$(COMPOSE_PROJECT_NAME)" info:)
+	$(eval CONTAINERS = $(shell docker ps -f name=$(COMPOSE_PROJECT_NAME) --format "{{ .ID }}"))
+	$(foreach CONTAINER, $(CONTAINERS),$(info $(shell printf '%-19s \n'  $(shell docker inspect --format='{{.NetworkSettings.Networks.$(COMPOSE_NET_NAME).IPAddress}} {{range $$p, $$conf := .NetworkSettings.Ports}}{{$$p}}{{end}} {{.Name}}' $(CONTAINER) | awk '{print $0}')) ))
+	@echo "$(RESULT)"
 
-## Run shell in PHP container as CUID:CGID user
+## Run shell in PHP container as regular user
 exec:
 	docker-compose exec --user $(CUID):$(CGID) php ash
 
-## Run shel in PHP container as root
+## Run shell in PHP container as root
 exec0:
 	docker-compose exec php ash
 
-up: net
-	@echo "Updating containers..."
-	docker-compose pull
-	@echo "Build and run containers..."
-	docker-compose up -d --remove-orphans
-
 down:
+	$(eval SCAFFOLD = $(shell docker-compose exec -T --user $(CUID):$(CGID) php composer run-script list-scaffold-files | awk '{if(NR>1)print}'))
 	@echo "Removing network & containers for $(COMPOSE_PROJECT_NAME)"
 	@docker-compose down -v --remove-orphans
 
+DIRS = web/core web/libraries web/modules/contrib web/profiles/contrib web/sites web/themes/contrib vendor
+
 ## Totally remove project build folder, docker containers and network
-clean: DIRS := core libraries modules/contrib profiles/contrib sites themes/contrib
 clean: info down
-	@for i in $(DIRS); do if [ -d "web/$$i" ]; then echo "Removing web/$$i..."; docker run --rm -v $(shell pwd):/mnt $(IMAGE_PHP) sh -c "rm -rf /mnt/web/$$i"; fi; done
+	@for i in $(DIRS); do if [ -d "$$i" ]; then echo "Removing $$i..."; docker run --rm -v $(shell pwd):/mnt $(IMAGE_PHP) sh -c "rm -rf /mnt/$$i"; fi; done
+	@for i in $(SCAFFOLD); do if [ -e "web/$$i" ]; then echo "Removing web/$$i..."; rm -rf web/$$i; fi; done
 	@if [ -d $(MYSQL_DATADIR) ]; then echo "Removing mysql data $(MYSQL_DATADIR)..."; docker run --rm -v $(MYSQL_BASE_PATH):/mnt/2rm $(IMAGE_PHP) sh -c "rm -rf /mnt/2rm/$(COMPOSE_PROJECT_NAME)_mysql"; fi
```

Comment:
Need to replace `MYSQL_BASE_PATH` by `DB_DATA_DIR`

Reply:
Need to replace `MYSQL_BASE_PATH` by `DB_DATA_DIR` at https://github.com/skilld-labs/skilld-docker-container/pull/61/files#diff-fd00501a70ad7e293f861012bdb54164R19 too
Also, why don't we uncomment this line? I see in all projects we end up uncommenting it.